.php_cs

```diff
@@ -1,6 +1,8 @@
 <?php
 
 $finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->exclude('build')
+    ->exclude('node_modules')
```

Comment:
`DefaultFinder` picks up XML files for some reason, so finds stuff in here.

Ideally we'd just want to parse `.gitignore`.